openwisp_controller/geo/api/views.py

```diff
@@ -69,12 +70,22 @@ class GeoJsonLocationListPagination(GeoJsonPagination):
     page_size = 1000
 
 
+class GeoJsonLocationFilter(filters.FilterSet):
+    organization_slug = filters.CharFilter(field_name='organization__slug')
+
+    class Meta:
+        model = Location
+        fields = ['organization_slug']
+
+
 class GeoJsonLocationList(FilterByOrganizationManaged, generics.ListAPIView):
     queryset = Location.objects.filter(devicelocation__isnull=False).annotate(
         device_count=Count('devicelocation')
     )
     serializer_class = GeoJsonLocationSerializer
     pagination_class = GeoJsonLocationListPagination
+    filter_backends = [filters.DjangoFilterBackend]
```

Comment:
We should define the filter_backend in settings instead if we are going to use filters for other endpoints too

Reply:
Here we are adding filters for only one view. Maybe we can do that in future. But then we will need to ask users to add that in their settings otherwise it won't work.